CONTRIBUTING.md

```diff
@@ -52,11 +52,11 @@ You can contribute to Erlang/OTP by opening a Pull Request.
 
 Make sure you create a new branch for your pull request with `git checkout -b new-branch-name`.
 Give the branch a short but descriptive name, like `stdlib/lists-length-fix`.
-Never do your work directly on `maint` or `master`.
+Never do your work directly on `main` or `master`.
```

Comment:
`maint` is on purpose. It's short for "maintenance."

Reply:
Sorry! :sweat_smile: